spring-petclinic-messaging-emulator/src/main/resources/application.yml

```diff
@@ -1,5 +1,11 @@
-spring:
-  application:
-    name: messaging-emulator
-  # config:
-  #   import: optional:configserver:${CONFIG_SERVER_URL:http://localhost:8888/}
+spring:
+  application:
+    name: messaging-emulator
+  config:
+    import: optional:configserver:${CONFIG_SERVER_URL:http://localhost:8888/}
+  sql:
+    init:
+      mode: always
+      platform: hsqldb
+server:
```

Comment:
is this needed? It wasn't there in the previous version.

Reply:
if you want to test locally yes. when deployed using config repo can be changed to mysql